test/unit/Vault/Vault.t.sol

```diff
@@ -159,37 +159,13 @@ contract VaultTest is UnitBaseSetup {
   }
 
   /* ============ Claimer ============ */
-  /* ============ disableAutoClaim ============ */
-  function testDisableAutoClaimFalse() public {
-    bool disable = false;
-
-    vm.expectEmit(true, true, true, true);
-    emit AutoClaimDisabled(address(this), disable);
```

Comment:
Remove also the event at the top of the file: https://github.com/pooltogether/v5-vault/blob/839fa4b9815b270abf01531a96e7df02294c2e7a/test/unit/Vault/Vault.t.sol#L22